model/generate.go

```diff
@@ -316,13 +317,13 @@ func (b *batchTimeTasksAndVariants) tasksWithoutBatchTime(taskNames []string, va
 func (g *GeneratedProject) findTasksAndVariantsWithBatchTime() batchTimeTasksAndVariants {
 	res := newBatchTimeTasksAndVariants()
 	for _, bv := range g.BuildVariants {
-		if bv.BatchTime != nil || bv.CronBatchTime != "" {
+		if bv.BatchTime != nil || bv.CronBatchTime != "" || bv.Activate != nil {
```

Comment:
I think maybe I'm misunderstanding something here. Wouldn't checking `bv.Activate` apply in both the `true` and `false` cases? And wouldn't we want not to activate ever in the `false` case?

Reply:
> Wouldn't checking bv.Activate apply in both the true and false cases? 

Yep, that's why I'm checking if it doesn't equal nil. If it does equal nil, then it's not explicitly being activated or not activated, so it'll default to being activated if the variant is supposed to be (i.e. it's not a special case). 

> And wouldn't we want not to activate ever in the false case? 

You're right that in the false case we never want to activate; I'm adding these variants to the version with ActivateAt set to the zero time, to ensure that we don't activate it unless it's explicitly set. If I don't specify this, then later when we activate everything that isn't in batch time info. I'm going to rename these things to make it clear that this isn't just batchtime now.